erts/emulator/beam/dist.c

```diff
@@ -5811,10 +5929,26 @@ BIF_RETTYPE erts_internal_dist_spawn_request_4(BIF_ALIST_4)
                         monitor_opts_oflags |= ERTS_ML_FLG_SPAWN_MONITOR;
                         add_monitor = 1;
                     }
-                    
+
+                    if (0) {
```

Comment:
The pattern of `if (0)` is either the cleverest or ugliest thing I've seen in a while, I can't decide which.

Reply:
I like it since you can get a common ending for multiple `case`s in a `switch` without having to resort to `goto`s. I'm not fully consistent with my use of this and `goto`s, though. Sometimes I use `goto`s...